<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNewLevelDialog</class>
 <widget class="QDialog" name="CNewLevelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>385</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="USE_TERRAIN">
     <property name="title">
      <string>Use Terrain</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="TERRAIN_RESOLUTION_LABEL">
        <property name="text">
         <string>Heightmap Resolution:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="buddy">
         <cstring>TERRAIN_RESOLUTION</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="TERRAIN_RESOLUTION"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="STATIC4">
        <property name="text">
         <string>Meters Per Texel:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="buddy">
         <cstring>TERRAIN_UNITS</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="TERRAIN_UNITS"/>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="TERRAIN_INFO">
        <property name="text">
         <string>Terrain Size: 32x32 Km</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QLabel" name="STATIC_WARNING">
        <property name="text">
         <string>Warning: After terrain resize, undo queue will be lost</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="TERRAIN_TYPE"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="TERRAIN_SIZE_LABEL">
        <property name="text">
         <string>Terrain Size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QFormLayout" name="formLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="TERRAIN_SIZE_X_LABEL">
          <property name="text">
           <string>x:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="TERRAIN_SIZE_Y_LABEL">
          <property name="text">
           <string>y:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="TERRAIN_SIZE_Z_LABEL">
          <property name="text">
           <string>z:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="TERRAIN_SIZE_X"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="TERRAIN_SIZE_Y"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="TERRAIN_SIZE_Z"/>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="STATIC_GROUP1">
     <property name="title">
      <string>Level</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="STATIC2">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="buddy">
         <cstring>LEVEL</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LEVEL">
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="STATIC1">
        <property name="text">
         <string>Folder:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="buddy">
         <cstring>LEVEL_FOLDERS</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="LEVEL_FOLDERS"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CNewLevelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CNewLevelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
